core/src/main/java/com/google/googlejavaformat/java/JavaFormatterOptions.java

```diff
@@ -30,6 +30,7 @@
 public class JavaFormatterOptions {
 
   static final int DEFAULT_MAX_LINE_LENGTH = 100;
+  static final String DEFAULT_JAVA_SOURCE_LEVEL = "9";
```

Comment:
This doesn't need configuration. It would normally default to the latest supported language level, which is want we want. Explicitly setting `-source 9` is a temporary work-around (see the TODO in my other comment).